salt/modules/grains.py

```diff
@@ -1,6 +1,13 @@
 # -*- coding: utf-8 -*-
 '''
 Return/control aspects of the grains data
+
+Grains set or altered this way are stored in the 'grains'
+file on the minions, by default at: ``/etc/salt/grains``
```

Comment:
I think this line could be more clear by splitting it up. Maybe something like `file on the minions. By default, this file is located at...`.